### ISSUE TITLE
Outgunned Compediums

### ISSUE THREAD

**Issue by jstruck80** [superseva/outgunned#3]
Not really an issue, but figured this was the easiest way to contact you about this. I am slowly building compendiums from all three books of the Outgunned kickstarter: Outgunned Core, Action Flicks, and World of Killers. I'm including Roles and Tropes as journal entries, hero and enemy feats, enemy special actions, and will try to do weapons and maybe some spells. When I'm done, I can share with you if you wanted to include them in the system you have graciously designed. Just let me know!

**Comment by superseva** [comment]
Hi there,
I am not sure we are allowed to do that. Those are copyrighted things and I don't have permission to add those with the system

**Comment by 9littlebees** [comment]
Indeed, including material from the books would not be allowed and this issue should probably be closed.
